cypress/page/iframe.page.ts

```diff
@@ -0,0 +1,32 @@
+class Iframe{
+    private URL: string;
+    private iFrameSelector: string;
+    private iFrameCssButtonSelector: string;
+
+    constructor(){
+        this.URL = "https://www.w3schools.com/html/html_iframe.asp"
+        this.iFrameSelector = "#main h1"
+        this.iFrameCssButtonSelector = "a[title='CSS Tutorial']"
+    }
+
+    public visit(): void{
+        cy.visit(this.URL)
+      }
+      
+    public getFrameTitle(titleElements: string): void{
+        cy.iframe().find(this.iFrameSelector).should("contains.text",titleElements)
+      }
```

Comment:
in this case, as we are asking to call the method as get, it should not perform any validation, you just should return the title of the iframe

Reply:
should i print it in cypress interface? Or just return it and store it in a variable?